coldfront/config/local_settings.py.sample

```diff
@@ -264,6 +264,22 @@ LOCAL_SETTINGS_EXPORT += [
 # ColdFront Plugin Settings
 #===============================================================================
 
+#------------------------------------------------------------------------------
+# Enable django debug toolbar
+#------------------------------------------------------------------------------
+EXTRA_APPS += [
+    'debug_toolbar'
+]
+
+EXTRA_MIDDLEWARE += [
+    'debug_toolbar.middleware.DebugToolbarMiddleware'
+]
+
+INTERNAL_IPS = [
```

Comment:
Can you add a brief comment noting that these are the IP addresses of clients that are allowed to view the toolbar?